README.md

```diff
@@ -126,7 +128,18 @@ Dr. OtterAI will:
 1. Review your code changes
 2. Add detailed comments
 3. Create a new PR with suggested fixes (coming soon)
+  > This will be a new PR that has the fixes
 4. Link the fix PR to your original PR (coming soon)
+  > This will be a link to the new PR that has the fixes
```

Comment:
🔗 Clarify what types of links should be included. This will help users understand how to effectively link their PRs.